CHANGELOGS.md

```diff
@@ -26,142 +23,144 @@ This document shows the history of when a specific sample was introduced to the
+|2021.1.Gold|[Vector Add DPCT](https://github.com/oneapi-src/oneAPI-samples/tree/master/Tools/Migration/vector-add-dpct)|Simple project to illustrate the basic migration of CUDA code. Use this sample to ensure your environment is configured correctly and to understand the basics of migrating existing CUDA projects to Intel® oneAPI Data Parallel C++ (DPC++)|
 |2021.1.Gold|[Vectorize VecMatMult](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/Fortran/DenseLinearAlgebra/vectorize-vecmatmult)|Fortran Tutorial - Using Auto Vectorization|
-|2021.1.Gold|[Intel® Python XGBoost Daal4py Prediction](https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/IntelPython_XGBoost_daal4pyPrediction)|This sample code illustrates how to analyze the performance benefit of minimal code changes to port pre-trained XGBoost model to daal4py prediction for much faster prediction than XGBoost prediction|
-|2021.1.Gold|[Particle Diffusion](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++/StructuredGrids/particle-diffusion)|The Particle Diffusion code sample illustrates Intel® oneAPI Data Parallel C++ (DPC++) using a simple (non-optimized) implementation of a Monte Carlo Simulation of the Diffusion of Water Molecules in Tissue|
-|2021.1.Gold|[Tutorials](https://github.com/oneapi-src/oneAPI-samples/tree/master/Libraries/oneDNN/tutorials)|Intel® oneDNN Tutorials|
-|2021.1.Gold|[CRR](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++FPGA/ReferenceDesigns/crr)|This sample shows a Binomial Tree Model for Option Pricing using a FPGA-optimized reference design of the Cox-Ross-Rubinstein (CRR) Binomial Tree Model with Greeks for American exercise options|
-|2021.1.Gold|[DPC Reduce](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++/ParallelPatterns/dpc_reduce)|This sample models transform Reduce in different ways showing capability of Intel® oneAPI|
+|2021.1.Gold|[AWS Pub Sub](https://github.com/oneapi-src/oneAPI-samples/tree/master/Tools/IoTConnectionTools/aws-pub-sub)|This sample uses the Message Broker for AWS* IoT to send and receive messages through an MQTT connection|
+|2021.1.Gold|[Zero Copy Data Transfer](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/zero_copy_data_transfer)|An Intel® FPGA tutorial demonstrating zero-copy host memory using the SYCL restricted Unified Shared Memory (USM) model|
+|2021.1.Gold|[Base: Vector Add](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC++/DenseLinearAlgebra/vector-add)|This simple sample adds two large vectors in parallel. Provides a ‘Hello World!’ like sample to ensure your environment is setup correctly using simple Intel® oneAPI Data Parallel C++ (DPC++)|
```

Comment:
Why are the last 9 out of order?

Reply:
There was an error in guids.json - lack of first letters in the name